samples/benchmarks/coremark/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Common coremark configuration
+CONFIG_COREMARK=y
+
+CONFIG_COREMARK_ITERATIONS=2000
+
+# Single thread execution
+CONFIG_COREMARK_THREADS_NUMBER=1
+# Choose memory method
+CONFIG_COREMARK_MEMORY_METHOD_STACK=y
+#CONFIG_COREMARK_MEMORY_METHOD_MALLOC=y
+
+# Supported only for single thread. No stack or heap required
+#CONFIG_COREMARK_MEMORY_METHOD_STATIC=y
+
+# Choose proper memory size based on memory method selected above
+# For 1 thread
+CONFIG_MAIN_STACK_SIZE=4096
+#CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+# For 4 threads
+CONFIG_TIMESLICE_SIZE=10
+#CONFIG_MAIN_STACK_SIZE=10240
+#CONFIG_HEAP_MEM_POOL_SIZE=10240
+
+# Compiler options
+CONFIG_COMPILER_OPT="-O3"
+
+# Config results output
+CONFIG_LOG=y
+CONFIG_PRINTK=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
```

Comment:
It is "nice to have" but not a crucial requirement, so lets leave it as it is.

Reply:
It will print integers instead. We loose a bit of accuracy ( up to +9% of score) but I wouldn't say it is a crucial issue. In fact we always have to make some approximation.

Reply:
I think we should select it (or at least imply). Silently printing result with significantly worse accuracy is not a good idea.

Reply:
Or you can set the CoreMark definition determining if we use floats for printing results or not relying on this Kconfig option